### PR TITLE
feat(client-debugger): add timestamp to log event

### PR DESCRIPTION
**Pull request by chentong7**
## Description

- Summary

The telemetry events generated by Fluid don't have a timestamp on them. When displaying them in the debugger, we should also print the timestamp for when they were received.

- Acceptance Criteria:

Each telemetry event displayed in the fluid debugger has an accompanying timestamp that indicates when it was received by the debugger

### CODE REVIEW

File: packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts [hunk 1]
```diff
@@ -8,6 +8,22 @@ import { IDebuggerMessage } from "./Messages";
 
 // #region Outbound messages
 
+/**
+ * Base interface for telemetry log events.
```
Comment (alexvy86) [inline packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts:12]:
```suggestion
 * Interface for telemetry events with a timestamp. Specific to the Fluid Debugger.
```

File: packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts [hunk 2]
```diff
@@ -8,6 +8,22 @@ import { IDebuggerMessage } from "./Messages";
...
+ * @public
+ */
+export interface TelemetryLogEvent {
```
Comment (alexvy86) [inline packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts:15]: Let's follow the convention of naming interfaces with an `I` at the beginning, and I suggest this other name.
```suggestion
export interface ITimestampedTelemetryEvent {
```

**Review by alexvy86 — APPROVED** [review]
A couple of small non-functional comments, but otherwise looks good!

File: packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts [hunk 3]
```diff
@@ -8,6 +8,22 @@ import { IDebuggerMessage } from "./Messages";
...
+ * @public
+ */
+export interface ITimestampedTelemetryEvent {
+	/**
+	 * The contents of the telemetry event.
+	 */
+	logContent: ITelemetryBaseEvent;
+
+	/**
+	 * The timestamp of the telemetry event.
```
Comment (Josmithr) [inline packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts:22]: Can we document the expected format for this time stamp?

Comment (Josmithr) [reply packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts:22]: We can document it in terms of `Date.now` if that's how we're populating it.

**Review by Josmithr — APPROVED** [review]

File: packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts (hunk 3 above)
Comment (chentong7) [reply packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts:22]: Added the example format and {@link Date.now()}

**Pull request merged**